jobs_detector/main.py

```diff
@@ -5,29 +5,81 @@
 from jobs_detector import settings
 
 DEFAULT_KEYWORDS = [
-    # set some default keywords here
+    'Remote', 'Python', 'Postgres', 'Javascript', 'React', 'Pandas'
 ]
 
 
-@click.group()
+#@click.group()
 def jobs_detector():
     pass
-
+"""
 
 @jobs_detector.command()
 @click.option('-i', '--post-id', type=str, required=True)
 @click.option('-k', '--keywords', type=str, default=','.join(DEFAULT_KEYWORDS))
 @click.option('-c', '--combinations', type=str,
               callback=lambda _, x: x.split(',') if x else x)
+"""              
 def hacker_news(post_id, keywords, combinations):
     """
     This subcommand aims to get jobs statistics by parsing "who is hiring?"
     HackerNews posts based on given set of keywords.
     """
-    # HINT: You will probably want to use the `BeautifulSoup` tool to
-    # parse the HTML content of the website
-    pass
+    stats = []
+    
```

Comment:
By commenting the decorators, you are trying to test the code, I assume. In that case, since the functions being called are defined 'after' the calling,  would this work?